homeassistant/components/onewire/onewirehub.py

```diff
@@ -66,13 +72,29 @@ def _discover_devices_owserver(self):
         """Discover all owserver devices."""
         devices = []
         for device_path in self.owproxy.dir():
-            devices.append(
-                {
-                    "path": device_path,
-                    "family": self.owproxy.read(f"{device_path}family").decode(),
-                    "type": self.owproxy.read(f"{device_path}type").decode(),
-                }
-            )
+            dtype = self.owproxy.read(f"{device_path}type").decode()
+            if dtype == MICROLAN_TYPE:
+                for branch in MICROLAN_BRANCHES:
+                    for sub_device_path in self.owproxy.dir(device_path + branch):
+                        devices.append(
+                            {
+                                "path": sub_device_path,
+                                "family": self.owproxy.read(
+                                    f"{sub_device_path}family"
+                                ).decode(),
+                                "type": self.owproxy.read(
+                                    f"{sub_device_path}type"
+                                ).decode(),
+                            }
+                        )
```

Comment:
Please add an argument to `_discover_devices_owserver`, and do a recursive call (your could theoretically chain them right?)
```suggestion
                for branch in MICROLAN_BRANCHES:
                    devices.append(self._discover_devices_owserver(device_path + branch))
```